book/http.md

```diff
@@ -553,16 +552,16 @@ endings.
     `Content-Type` header usually contains a `charset` declaration
     that specifies the encoding of the body. If it's absent, browsers
     still won't default to `utf8`; they'll guess, based on letter
-    frequencies, and you see ugly � strange áççêñ£ß when they guess
-    wrong. Incorrect-but-common `utf8` skips all that complexity.
```

Comment:
I don't know what this last sentence means

Reply:
I think it means "`utf8` is common enough that our browser assuming it works well enough for our purposes." Let's leave this sentence in.